index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="menu-bar">
+      <a class="name" href="#">Eyob yoseph</a>
+      <img class="humburger" src="img/Icon - Menu.png" alt="">
+    </nav>
+  </header>
```

Comment:
Great job implementing the toolbar 👍. Kindly remove the logo from nav and let it stand on its own e.g
```
<header>
  <a class="name" href="#">Eyob yoseph</a>
    <nav class="menu-bar">
      <img class="humburger" src="img/Icon - Menu.png" alt="">
    </nav>
  </header>
```

If you like, you can wrap it inside a `div`. Removing it from the nav is because of HTML semantic usage 😉 . You can check this[ link ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)or more explanation.

Reply:
I have done  as I  am advised to do